scripts/check-toffee-plum.sh

```diff
@@ -4,6 +4,7 @@ function add_environments() {
     if [[ "$1" == "Toffee" ]]; then
         ENVIRONMENTS=("Sandbox" "Test" "ITHC" "Demo" "Staging" "Prod")
     fi
+
```

Comment:
Are able to please remove all the extra whitespace / newlines added in this? Few other spots too